src/helm/benchmark/scenarios/bhasa_scenario.py

```diff
@@ -0,0 +1,1908 @@
+
+# BHASA Scenarios
+#   A. Natural Language Understanding
+#   B. Natural Language Generation
+#   C. Natural Language Reasoning
+#   D. Linguistic Diagnostics
+
+# A. Natural Language Understanding
+#   1. Question Answering
+#   2. Sentiment Analysis
+#   3. Toxicity Detection/Classification
+
+# 1. Question Answering
+# 1.1 Indonesian: TyDiQA
+class TyDiQAScenario(Scenario):
+    """
+    TyDiQA is is an open-book question answering scenario for 11 typologically-diverse languages.
+    The questions are written by people who want to know the answer, but do not know the answer yet,
+    and the data is collected directly in each language without the use of translation.
+
+    This scenario only uses the Indonesian subset of the data, and uses the Gold Passage (GoldP) task,
+    which requires the tested system to extract a span from the given passage to answer a given question.
+    There are no unanswerable questions.
+
+    The models are prompted using the following format:
+
+        Anda akan diberikan sebuah paragraf dan sebuah pertanyaan. Jawablah pertanyaannya dengan mengekstrak jawaban dari paragraf tersebut.
+
+        Paragraf: <text>
+        Pertanyaan: <question>
+        Jawaban: <answer>
+
+        ...
+
+        Paragraf: <text>
+        Pertanyaan: <question>
+        Jawaban:
+
+
+    Target completion:
+        <answer>
+
+    @article{clark-etal-2020-tydi,
+        title = "{T}y{D}i {QA}: A Benchmark for Information-Seeking Question Answering in Typologically Diverse Languages",
+        author = "Clark, Jonathan H.  and
+        Choi, Eunsol  and
+        Collins, Michael  and
+        Garrette, Dan  and
+        Kwiatkowski, Tom  and
+        Nikolaev, Vitaly  and
+        Palomaki, Jennimaria",
+        editor = "Johnson, Mark  and
+        Roark, Brian  and
+        Nenkova, Ani",
+        journal = "Transactions of the Association for Computational Linguistics",
+        volume = "8",
+        year = "2020",
+        address = "Cambridge, MA",
+        publisher = "MIT Press",
+        url = "https://aclanthology.org/2020.tacl-1.30",
+        doi = "10.1162/tacl_a_00317",
+        pages = "454--470",
+    }
+    """
+
+    name = "tydiqa"
+    description = "Indonesian Open-book Question Answering task"
+    tags = ["question_answering"]
+
+    def __init__(self):
+        super().__init__()
+        self.splits = {
+            "train": TRAIN_SPLIT,
+            "validation": TEST_SPLIT
+        }
+
+    def get_instances(self, output_path) -> List[Instance]:
+        dataset = datasets.load_dataset("khalidalt/tydiqa-goldp", "indonesian")
+
+        outputs = []
+        for split in self.splits.keys():
+            df = dataset[split].to_pandas()
+
+            if split == "train":
+                # Select only bottom 20th percentile by length for in-context examples as examples are very long
+                data = df[df["passage_text"].apply(len) < df["passage_text"].apply(len).quantile(.2)]
```

Comment:
Is there possible overlap between the train and test splits? I noticed `XQuADScenario` has a fix that avoids this - is this also needed here?

Reply:
TyDiQA does have a train split and validation split, so we use the train for train and validation for test.